tests/tests/test_mtls.py

```diff
@@ -133,15 +133,13 @@ def hsm_get_key_uri(self, pin, ssl_engine_id, device):
             + '\nMODULE_PATH = /usr/lib/softhsm/libsofthsm2.so\ninit = 0\n" >> /etc/ssl/openssl.cnf'
         )
         device.run(
-            'sed -i.backup -e "/\\[Service\\]/ a Environment=SOFTHSM2_CONF=/softhsm/softhsm2.conf" /lib/systemd/system/%s.service'
-            % device.get_client_service_name()
+            'sed -i.backup -e "/\\[Service\\]/ a Environment=SOFTHSM2_CONF=/softhsm/softhsm2.conf" /lib/systemd/system/mender-authd.service'
```

Comment:
This now, as far as I can tell, needs to be done for both services :thinking:

Reply:
I'll do it for good measure. This test is not passing yet.